segmentio/analytics-ios
# Added Swift Package Manager support

**What does this PR do?**
Adds Swift Package Manager support and maintains support for Carthage, CocoaPods and manual building.

**Where should the reviewer start?**
Verify the filesystem structure is accurate and acceptable.

**How should this be manually tested?**
Add the package to a new Xcode project. Then verify the next tag works for CocoaPods in a new sample project. Finally check the project works with Carthage as well, using the latest tag.

**Any background context you want to provide?**
SPM tests had to be removed. The normal tests can be used via CI to verify our projects.

**What are the relevant tickets?**
LIBMOBILE-74

**Screenshots or screencasts (if UI/UX change)**

**Questions:**
- Does the docs need an update?
Yes

- Are there any security concerns?
No

- Do we need to update engineering / success?
